alsaaudio.c

```diff
@@ -2411,14 +2411,14 @@ alsamixer_getvolume(alsamixer_t *self, PyObject *args, PyObject *kwds)
 
 	elem = alsamixer_find_elem(self->handle,self->controlname,self->controlid);
 
-	if (!pcmtypeobj || (pcmtypeobj == Py_None)) {
+	if (!pcmtypeobj || (pcmtypeobj == Py_None))
```

Comment:
Can you adjust the identation to the style use here? Like this?
```
	if (!pcmtypeobj || (pcmtypeobj == Py_None)) {
		if (self->pchannels) {
			pcmtype = SND_PCM_STREAM_PLAYBACK;
		} 
		else {
			pcmtype = SND_PCM_STREAM_CAPTURE;
		}
	}
```

Reply:
Fixed.